tools/docker/tests/test_gw_repeater.sh

```diff
@@ -40,7 +40,7 @@ main() {
             -d | --delay)         DELAY="$2"; shift; shift ;;
             -f | --force)         FORCE_OPT="-f"; shift ;;
             -g | --gateway)       GW_NAME="$2"; shift; shift ;;
-            -r | --repeater)      REPEATER_NAME="$2"; shift; shift ;;
+            -r | --repeater)      REPEATER_NAMES="$2 $REPEATER_NAMES"; shift; shift ;;
```

Comment:
If you set this to 
```suggestion
            -r | --repeater)      REPEATER_NAMES="$REPEATER_NAMES $2"; shift; shift ;;
```
you don't need to reverse them.